compiler/common-artifacts/exclude.lst

```diff
@@ -9,6 +9,7 @@ optimize(Add_STR_000) # STRING is not supported
 optimize(Add_STR_001) # STRING is not supported
 
 ## CircleRecipes
+optimize(RmsNorm_000)
```

Comment:
plz split this to another PR.

Reply:
> plz split this to another PR.

Okay, I'll submit another PR.

Reply:
It's merged as https://github.com/Samsung/ONE/pull/14032